src/common/createplay/CreatePlay.jsx

```diff
@@ -11,6 +11,21 @@ import Loader from "common/spinner/spinner";
 import { Plays } from "common/services/plays";
 import { useNavigate } from "react-router-dom";
 import { useSearchParams } from "react-router-dom";
+import { ReactComponent as NotAllowedImage } from "../../images/img-403.svg"
+
+const NoCreationInProdScreen = () => {
+  return <div className="w-full h-full flex flex-col justify-center items-center gap-8">
+    <div>
+      <NotAllowedImage width={550} />
+    </div>
+
+    <div className="text-center">
+      You can't create plays in production
+      <br />
+      <a className="text-link-default" href="https://github.com/reactplay/react-play#readme" target="__blank__">read this</a> for more details
```

Comment:
Shouldn't it point to the create play md? CREATE-PLAY.md

Reply:
done changed it 👍